modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -866,6 +879,13 @@ def test_drop_duplicates_after_sort():
     df_equals(modin_result, pandas_result)
 
 
+def test_drop_duplicates_with_repeated_index_values():
+    data = {"data": [[0], [1], [0]], "index": [0, 0, 0]}
+    pandas_df = pandas.DataFrame(data["data"], index=data["index"])
+    modin_df = pd.DataFrame(data["data"], index=data["index"])
+    df_equals(modin_df.drop_duplicates(), pandas_df.drop_duplicates())
```

Comment:
```suggestion
def test_drop_duplicates_with_repeated_index_values():
    data = [[0], [1], [0]]
    index = [0, 0, 0]
    modin_df, pandas_df = create_test_dfs(data, index=index)
    eval_general(modin_df, pandas_df, lambda df: df.drop_duplicates())
```

Applying some modin testing utilities like this should make the test clearer.